be/src/olap/base_tablet.cpp

```diff
@@ -1066,6 +1066,15 @@ void BaseTablet::add_sentinel_mark_to_delete_bitmap(DeleteBitmap* delete_bitmap,
     }
 }
 
+void BaseTablet::add_skip_mark_to_delete_bitmap(DeleteBitmap* delete_bitmap,
```

Comment:
you don't need to add a mark, just check the rowset's version, if the version is not single version, e.g [x-x], it means that the new rowset is generated by compaction, you can skip it directly